src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Consider using a constant for the number of days that qualifies for the long-term discount and the amount of the discount itself, as per the checklist item 2. This makes the code more readable and maintainable.